src/libLookup/Lookup.cpp

```diff
@@ -144,12 +232,13 @@ bool Lookup::GenTxnToSend(size_t num_txn,
 
   unsigned int NUM_TXN_TO_DS = num_txn / GENESIS_WALLETS.size();
 
+  if (numShards == 0) {
+    return false;
```

Comment:
Is it possible to give a warning message before returning false, to let user know what happened?